### PR TITLE
Bug fix

### PR DESCRIPTION
**Pull request by vrathnayake**
please check

### CODE REVIEW

File: test.txt [hunk 1]
```diff
@@ -3,3 +3,7 @@ Line2 k
 Line 2 tina
 Line 3 k
 Line 3 tina
+
+function = () =>{
```
Comment (kumudug) [inline test.txt:7]: Me mona wikarayakda me !

File: test.txt [hunk 2]
```diff
@@ -3,3 +3,7 @@ Line2 k
...
+ console.log();
```
Comment (kumudug) [inline test.txt]: Meeeka hari yanne ne

File: test.txt (hunk 1 above)
Comment (vrathnayake) [reply test.txt:7]: yanawa yanna

File: test.txt (hunk 2 above)
Comment (kumudug) [reply test.txt]: Meke mona pukak wath print wenne nane

Comment (vrathnayake) [reply test.txt]: aa huta!!!

Comment (vrathnayake) [reply test.txt]: fixed

Comment (kumudug) [reply test.txt]: Ehenan ona pukak kara ganna ekai atte.

**Pull request merged**